aws-samples/aws-tools-for-babylonjs-editor
# Update Package Names and Add Legal Info

## Description
Updated all of the package names to be the agreed upon names. Likewise, updated all references in code that pointed to old folder directory or was a customer visible label. Also, updated the "SumerianToolbar" class for each package to better reflect new name. 

Additionally, as part of this change, I've added some legal safeguards to the code:
 * Added NOTICE and LICENSE file to each sub-package
 * Updated header of all files to include appropriate license information
 * Updated package.json for each package to better represent author, description, and repo information

## Note
This change is built on top of [this PR](https://github.com/aws-samples/aws-tools-for-babylonjs-editor/pull/9), as I am unable to build successfully without. That change should be committed first before this one goes through. 

## Reviewer Testing Instructions
* Pull down changes
* Build each package, and verify that it works as it did before
* Add package to Babylon.JS Editor and verify it works as it did before; also verify that the text present looks correct for the UI modifications

## Testing done
* Successfully built each package locally via README instructions
* Successfully added each package to the Babylon.JS Editor and verified new text was present

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
